android/src/main/java/com/tailscale/ipn/ui/model/TailCfg.kt

```diff
@@ -91,6 +92,14 @@ class Tailcfg {
           Capabilities?.contains("https://tailscale.com/cap/is-admin") == true ||
               CapMap?.contains("https://tailscale.com/cap/is-admin") == true
 
+    // Derives the url to directly administer a node
+    val nodeAdminUrl: String
+      get() {
+        val address =
+                displayAddresses.firstOrNull { it.type == DisplayAddress.addrType.V4 }?.address
+        return if (address != null) "${Links.ADMIN_URL}/machines/${address}" else Links.ADMIN_URL
+      }
```

Comment:
displayAddresses already removes the mask and categorizes all of our IPs... The adminURL is the first ipV4.

Reply:
Might as well use this same logic for extracting the primary IP for display on the peer list.

Reply:
Done.  With a fallback to the ipV6 address...